documentation/asciidoc/topics/yaml/cloud_events.yaml

```diff
@@ -0,0 +1,8 @@
+spec:
+  cloudEvents:
+    # Configures the number of acks.
+    acks: "1"
```

Comment:
just for information, acks could be number or "all" value, thanks the reason why it's a string
https://betterprogramming.pub/kafka-acks-explained-c0515b3b707e

Reply:
nice one @dmvolod even though that might be kafka specific I've updated the comment with some additional detail

Reply:
@rigazilla could we add a validation for the acks values in CRD?